source/appModules/explorer.py

```diff
@@ -299,3 +300,16 @@ def event_gainFocus(self, obj, nextHandler):
 			return
 
 		nextHandler()
+
+	def isGoodUIAWindow(self,hwnd):
+		# #9204: In build 18305 and later, it is the shell that raises window open event for emoji panel.
+		if winVersion.winVersion.build > 17763 and winUser.getClassName(hwnd) == "ApplicationFrameWindow":
+			return True
+		return False
+
+	def event_UIA_window_windowOpen(self, obj, nextHandler):
+		# Send UIA window open event to input app window.
+		if isinstance(obj, UIA) and obj.UIAElement.cachedClassName == "ApplicationFrameWindow":
+			inputPanelWindow = obj.firstChild
+			if inputPanelWindow and inputPanelWindow.appModule.appName == "windowsinternal_composableshell_experiences_textinput_inputapp":
+				eventHandler.executeEvent("UIA_window_windowOpen", inputPanelWindow)

```

Comment:
Note that you no longer call nextHandler here. I guess that doesn't matter as it is very unlikely that there'll be a next handler in this particular case. Personally, I'd still add it though